packages/evolution-frontend/src/components/admin/pages/CorrectionPage.tsx

```diff
@@ -12,12 +12,12 @@ import { faFolderOpen } from '@fortawesome/free-solid-svg-icons/faFolderOpen';
 import { ThunkDispatch } from 'redux-thunk';
 
 import InterviewSummary from '../validations/InterviewSummary';
-import { startSetSurveyValidateInterview } from '../../../actions/SurveyAdmin';
+import { startSetSurveyCorrectedInterview } from '../../../actions/SurveyAdmin';
 import InterviewListComponent from '../validations/InterviewListComponent';
 import { RootState } from '../../../store/configureStore';
 import { SurveyAction } from '../../../store/survey';
 
-const ValidationPage = () => {
+const CorrectionPage = () => {
```

Comment:
I don't think this is the CorrectionPage, it's more the ReviewPage, no?

Reply:
renamed